helm/cluster-cloud-director/values.yaml

```diff
@@ -94,10 +94,14 @@ nodePools: {}
   #   replicas: 2
 
 ssh:
-  users:  # (Optional) public SSH keys/users to access the nodes.
-    - name: ""
-      authorizedKeys:
-        - ""
+  trustedCaKeys:
+  # Taken from https://vault.operations.giantswarm.io/v1/ssh/public_key
+  - ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIM4cvZ01fLmO9cJbWUj7sfF+NhECgy+Cl0bazSrZX7sU vault-ca@vault.operations.giantswarm.io
```

Comment:
Are we going to introduce a validation rule for that?

Reply:
Puja said it is OK
https://gigantic.slack.com/archives/C01F7T2MNRL/p1674655810773929